reference/docs-conceptual/components/console/PowerShell.exe-Command-Line-Help.md

```diff
@@ -1,167 +1,15 @@
 ---
-ms.date:  08/14/2018
+ms.date:  05/02/2019
 keywords:  powershell,cmdlet
-title:  PowerShell.exe Command Line Help
-ms.assetid:  1ab7b93b-6785-42c6-a1c9-35ff686a958f
+title:  PowerShell command line options
```

Comment:
With the title change, should the file name be updated?

Reply:
Didn't want to deal with the redirect. This may change in a planned reorg so I will leave it alone for now .